shotover-proxy/src/message/mod.rs

```diff
@@ -354,6 +355,33 @@ pub enum IntSize {
     I8,  // Tinyint
 }
 
+impl From<&Value> for SQLValue {
+    fn from(v: &Value) -> Self {
+        match v {
+            Value::NULL => SQLValue::Null,
+            Value::Bytes(b) => SQLValue::SingleQuotedString(String::from_utf8(b.to_vec()).unwrap()), // TODO: this is definitely wrong
```

Comment:
It might be worth fixing this here? Or given a likely change in what we do with C* AST ... we can punt down the road again

Reply:
Yeah I would rather address this properly later on.

After all this change is equivalent to just making these methods public.
But calling them from CassandraCodec::expr_to_value was not ideal.